src/plugins/intel_cpu/src/infer_request.h

```diff
@@ -56,7 +56,9 @@ class InferRequestBase : public InferenceEngine::IInferRequestInternal {
     virtual void PushInputData() = 0;
 
     Graph* graph = nullptr;
-    std::unordered_map<std::string, void*> externalPtr;
+    std::vector<std::string> inplacedOutPorts;
+    std::vector<std::string> externalInPorts;
+    std::vector<std::string> externalOutPorts;
 
```

Comment:
There is no need to use externalPtr, use externalInPorts and externalOutPorts to keep port names instead, as we can access blob from _inputs and _outputs of InferRequestBase class.
externalPtr would make syncing between InferRequest and Graph more troublesome. So I replaced it.